app/assets/javascripts/modules/show_hide.js.coffee

```diff
@@ -0,0 +1,52 @@
+define('show_hide', [], ->
+  class ShowHide
+    constructor: (@$button) ->
+      @setup()
+      @addEventListeners()
+
+    setup: () ->
+      $wrappers = @$button.parents('.js-show-hide-wrapper')
+
+      $.each($wrappers, (index, wrapper) =>
+        $wrapper = $(wrapper)
+
+        $cards = $wrapper.find('.js-show-hide-target')
+
+        @hideCards($cards)
+
+        if($cards.length < 7) 
```

Comment:
Can we define a constant variable instead of using a plain integer?
I think it can also come in handy in a piece of code below when you do the slice